app/Resources/views/article.html.twig

```diff
@@ -0,0 +1,8 @@
+<html>
+    <head>
+        <title>Superdesk Web Publisher</title>
+    </head>
+    <body>
+        <h3>This is default article.html.twig template. Please replace it by creating your own version in used theme (views/article.html.twig).</h3>
```

Comment:
`This is default "article.html.twig" template file. Please, replace it by creating your own version of this file in the currently used theme.` sounds much clearer.`